src/defaultsettings.cpp

```diff
@@ -54,6 +54,7 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("curl_file_download_timeout", "300000");
 	settings->setDefault("curl_verify_cert", "true");
 	settings->setDefault("enable_remote_media_server", "true");
+	settings->setDefault("max_processed_meshes_per_frame", "5");
```

Comment:
Yes it's not a tuned value, it is set to an extreme value to make any improvement more noticeable, to see if this method helps.